client/web/dev-sandbox/public/static/misc/ftml-test-cases-export.ftml

```diff
@@ -15,11 +15,6 @@ Durian
 [[/>]]
 [[/=]]
 
-[!-- align-fail --]
```

Comment:
Is this / should this be marked as generated?

Reply:
Probably? It's probably not worth it hardcoding into the git attributes because the dev sandbox is an intentionally volatile module

Reply:
It's volatile? What does this module do exactly?

Reply:
It's literally a playground website I'm currently testing the editor on. It's in the repo because it does take a good bit of setup to use and it's generally useful for everyone to have access to. It also actually makes sure that the monorepo can build a website